### ISSUE TITLE
Unhandled exception when querying AMD OpenCL

### ISSUE THREAD

**Issue by josephno1** [nicehash/NiceHashMiner-Archived#18]
Hey I am getting this error on the loading screen when it hits the AMD OpenCL part. Here is a picture of the error message. I am running an AMD Gpu and I have 15.7.1 drivers.
![amd](https://cloud.githubusercontent.com/assets/15186942/12069255/73c618c8-afda-11e5-87c8-80b074cd1a08.PNG)

**Comment by Tmerlo** [comment]
I'm also getting an Exception error during "loading, please wait..."

```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.ComponentModel.Win32Exception: The system cannot find the file specified
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at NiceHashMiner.ccminer.QueryCDevs()
   at NiceHashMiner.ccminer_sp..ctor()
   at NiceHashMiner.Form1.StartupTimer_Tick(Object sender, EventArgs e)
   at System.Windows.Forms.Timer.OnTick(EventArgs e)
   at System.Windows.Forms.Timer.TimerNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.8670 (QFE.050727-8600)
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v2.0.50727/mscorlib.dll
----------------------------------------
NiceHashMiner
    Assembly Version: 1.2.2.0
    Win32 Version: 1.2.2.0
    CodeBase: file:///C:/Users/Anthony/Downloads/NiceHashMiner_v1.2.2.0/NiceHashMiner_v1.2.2.0/NiceHashMiner.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.8670 (QFE.050727-8600)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Windows.Forms/2.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.8670 (QFE.050727-8600)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System/2.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.8670 (QFE.050727-8600)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Drawing/2.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Management
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.8670 (QFE.050727-8600)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Management/2.0.0.0__b03f5f7f11d50a3a/System.Management.dll
----------------------------------------
Newtonsoft.Json
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.1.18622
    CodeBase: file:///C:/Users/Anthony/Downloads/NiceHashMiner_v1.2.2.0/NiceHashMiner_v1.2.2.0/Newtonsoft.Json.DLL
----------------------------------------
log4net
    Assembly Version: 1.2.15.0
    Win32 Version: 1.2.15.0
    CodeBase: file:///C:/Users/Anthony/Downloads/NiceHashMiner_v1.2.2.0/NiceHashMiner_v1.2.2.0/log4net.DLL
----------------------------------------
System.Configuration
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.8670 (QFE.050727-8600)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Configuration/2.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Xml
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.8670 (QFE.050727-8600)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Xml/2.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
```

**Comment by Tmerlo** [comment]
Had the same problem. Make sure Malwarebytes and any other virus scanners have the folder added to the exception list. Malwarebytes kept deleting ccminer and sgminer on me.

**Comment by tonyrayo** [comment]
Dupe of issue #14 
Edit: I should say "possible dupe".  Before now I thought that this was only an issue for non-AMD users.  I'll be debugging the program tomorrow, can you tell me what OS you are running (assuming it wasn't a virus scanner deleting exe files)?

**Comment by Unerion** [comment]
I have the same problem here.
Using Windows 7 Pro 64Bit
Also with the updated Version from #14 didnt help

Virus Scanner is not deleting any exe files and the directory is excluded from scanning.

**Comment by josephno1** [comment]
@tonyrayo 
I have all antiviruses disabled and I am running an AMD card with Windows 10 64 bit

**Comment by theLosers106** [comment]
@josephno1 hi.. could you download this (link removed) and replace the NiceHashMiner.exe .. please report your findings.. thanks (:

**Comment by Unerion** [comment]
downloaded the version and it works perfectly for me now =)

**Comment by kenshirothefist** [comment]
Resolved in v1.2.2.2.
